ddsketch/store/store.go

```diff
@@ -14,5 +18,8 @@ type Store interface {
 	MinIndex() (int, error)
 	TotalCount() float64
 	KeyAtRank(rank float64) int
+	KeyAtDescendingRank(rank float64) int
```

Comment:
This should be equal to `KeyAtRank(TotalCount() - 1 - rank)`. Or do we need it because of the lower/upper rank distinction? If so, we could also add a parameter to `KeyAtRank`.